Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -20,6 +20,12 @@ import Foundation
 import RealmSwift
 import XCTest
 
+private extension URL {
```

Comment:
This looks like an incorrect workaround for the source URL being constructed incorrectly.